packages/design-tokens/stories/ThemeColors.stories.tsx

```diff
@@ -26,37 +26,35 @@ export const FigmaLightTheme = {
     if (!lightTheme) {
       return null; // or some fallback component
     }
-    console.log('lightTheme', lightTheme);
     return <ColorSwatchGroup swatchData={lightTheme} />;
   },
+  parameters: {
+    colorScheme: 'light',
+  },
 };
 
 export const FigmaDarkTheme = {
   render: () => {
     const { darkTheme } = useJsonColor();
-    console.log('darkTheme', darkTheme);
     if (!darkTheme) {
-      return null; // or some fallback component
+      return null;
     }
-    return (
-      <div
-        style={{
-          margin: '-1rem',
-          padding: '1rem',
-        }}
-      >
-        <ColorSwatchGroup
-          swatchData={darkTheme}
-          theme={darkTheme?.background?.default?.value}
-        />
-      </div>
-    );
+
+    const backgroundColor =
+      'background' in darkTheme &&
+      typeof darkTheme.background === 'object' &&
+      darkTheme.background !== null &&
+      'default' in darkTheme.background &&
+      typeof darkTheme.background.default === 'object' &&
+      darkTheme.background.default !== null &&
+      'value' in darkTheme.background.default
+        ? darkTheme.background.default.value
+        : undefined;
+
+    return <ColorSwatchGroup swatchData={darkTheme} theme={backgroundColor} />;
   },
```

Comment:
Don't love how hard this is to read, but it fixes a type issue, and since this is just for Storybook and not an actual component, I think it’s fine for now.  

![Screenshot of the code fixing a type issue](https://github.com/user-attachments/assets/ae4eaf2e-6c94-4bfe-a767-67bb4b5f0cc3)

Reply:
holy conditioning